package.json

```diff
@@ -5,8 +5,9 @@
   "scripts": {
     "apps": "./script/app-list.sh",
     "analyze": "webpack-bundle-analyzer build/vagovprod/generated/stats.json",
-    "build": "./script/build.sh",
+    "build": "yarn build:webpack --env.scaffold",
```

Comment:
Might we still want to keep using `script/build.sh`? By switching back to directly using the `build:webpack` command, we'd be forcing devs back to using the `--env.*` flags.

Reply:
If there's no other use case for `build:clean`, this means the line in `jenkins/build.sh` could be refactored to still use this `build` command (note that `--scaffold` is removed since it would be "built-in"):

```sh
npm --no-color run build -- --verbose --buildtype="$envName" 2>&1 | tee "$buildLog"
```

Reply:
I hit some issues with that since it's also called through Jenkins, but I can look into it more